api/src/data/migrations/023.create-incident-logs.ts

```diff
@@ -5,16 +5,16 @@ export async function up(knex: knex.Knex) {
         table.increments("id").primary().notNullable();
         table.integer("incident_id").notNullable();
         table.string("old_description", 4096).notNullable();
-        table.string("old_sensitivity", 256).nullable();
+        table.string("old_sensitivity_code", 256).nullable();
         table.string("old_supervisor", 8).nullable();
```

Comment:
Is this an `id` column in the original table, or just a String?

Reply:
Also ... now that I looks at this, do these old_/new_ columns match the source table?
Might want to check that for all the "log" tables.

[Meld](https://meldmerge.org/) suggests some differences beyond the expected ones.
![image](https://github.com/icefoganalytics/safety/assets/23045206/ab4c82cc-8960-4dd8-934e-fb4150ea70f3)

Reply:
Looking at this again, we will need to either
a) remove the foreign key reference to `incidents.id`
b) add a `deleted_at` column to the `incidents` table

Why? Because we won't be able to track the history of deleted `incidents` as the foreign key relationship will restrict deletion.